bentoml/_internal/configuration/__init__.py

```diff
@@ -4,23 +4,17 @@
 import logging
 from functools import lru_cache
 
-from bentoml.exceptions import BentoMLException
-from bentoml.exceptions import BentoMLConfigException
+from ...exceptions import BentoMLException
+from ...exceptions import BentoMLConfigException
 
 try:
-    import importlib.metadata as importlib_metadata
-except ModuleNotFoundError:
-    import importlib_metadata
-
-import yaml
-
-try:
-    import bentoml._version as version_mod
+    from ..._version import __version__
+    from ..._version import version_tuple
```

Comment:
Since we are setting version inside `pyproject.toml` to be dynamic, we can do poor man  version compare here.

Reply:
I'm not sure what this means.

Reply:
Can you explain?

Reply:
I just realized that this change is not needed, but I think for the sake of simplicity, we should just import __version__ from _version instead of importing the module to fix types